src/modules/main/main-shell/Breadcrumbs.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { Breadcrumbs as MUIBreadcrumbs, Link, Typography } from '@mui/material';
+
+const Breadcrumbs = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const breadCrumbView = () => {
+    const { pathname } = location;
+    const pathnames = pathname.split("/").filter((item) => item);
+    const capatilize = (s) => s.charAt(0).toUpperCase() + s.slice(1);
+    return (
+      <div>
+        <MUIBreadcrumbs aria-label="breadcrumb">
+            {pathnames.length > 0 ? (
+            <Link onClick={() => navigate("/")}>Home</Link>
+            ) : (
+            <Typography>Home</Typography>
+            )}
```

Comment:
Codacy has a fix for the issue: [Replace `············` with `··········`](https://app.codacy.com/gh/abdulqhayum47/lets-react/pullRequest?prid=9776931)
```suggestion
          )}
```